### DISCUSSION TITLE
RFC's information of Strongswan 5.7.2

### DISCUSSION THREAD

**Discussion by ashutoshtekriwal2019** [strongswan/strongswan#1401] [General]
We are using Strongswan 5.7.2 for IPSec in our product(Encryptor Module). We are 
searching for proper page for RFC compliance claims. Following link 
"https://docs.strongswan.org/docs/5.9/features/ietf.html" talk about  RFC's that are atleast partial compliant by Strongswan. We want to know about list of RFC's that are fullycompliant. Kindly provide us the list. RFC's for this information is requested for our  side: 4301, 4303, 3602, 4106, 4868, 5282, 4945,7296.

**Comment by tobiasbrunner** [comment]
I don't know if strongSwan and the Linux kernel are fully-compliant to _all_ of these RFCs. But they are definitely NOT fully-compliant to RFC 4301 and RFC 7296, as there is always some detail that's missing (e.g. because there is just no use case for it), or will not be handled to the letter (e.g. to simplify the implementation or configuration). But all those RFCs are implemented to a degree that you probably won't notice any issues.
